model/docs/hospital_admissions_model.qmd

```diff
@@ -497,8 +535,112 @@ And plotting the results:
 out = hosp_model_weekday.plot_posterior(
     var="latent_hospital_admissions",
     ylab="Hospital Admissions",
-    obs_signal=np.pad(
-        dat_w_padding, (gen_int_array.size, 0), constant_values=np.nan
+    obs_signal=np.pad(dat_w_padding, (gen_int_array.size, 0), constant_values=np.nan),
+)
+```
+
+We will use ArviZ to visualize the posterior/prior predictive distributions.
+
+```{python}
+# | label: posterior-predictive-distribution
+idata_weekday = az.from_numpyro(
+    hosp_model_weekday.mcmc,
+    posterior_predictive=hosp_model_weekday.posterior_predictive(
+        n_timepoints_to_simulate=len(dat_w_padding)
+    ),
+    prior=hosp_model_weekday.prior_predictive(
```

Comment:
Just noticed that we are using `prior` for what should be `prior_predictive`. However, it doesn't look like `arviz.from_numpyro` has a `prior_predictive` argument. I have opened an issue https://github.com/arviz-devs/arviz/issues/2358

Reply:
From [this](https://python.arviz.org/en/stable/getting_started/CreatingInferenceData.html#from-numpyro) example, the `prior` argument in `arviz.from_numpyro` takes `prior_predictive` as an input.

Reply:
Ah, I see that supplying `prior` automatically creates the `prior_predictive` component of the `InferenceData` object.